src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -750,45 +744,16 @@ public static uint HeuristicDivide(ref BigInteger dividend, ref BigInteger divis
                 return quotient;
             }
 
-            public static uint LeadingZeroCount(uint value)
-            {
-                return 32 - CountSignificantBits(value);
-            }
-
-            public static uint LeadingZeroCount(ulong value)
-            {
-                return 64 - CountSignificantBits(value);
-            }
-
             public static uint LogBase2(uint value)
             {
```

Comment:
These `Log2Base` wrappers do not add much value. (And they do not really belong here because of they are not related to BigInteger.) Call them from where they are used?

Reply:
I have inlined the callsites, but not sure if I should remove this method from the public interface?

Reply:
Oh don't worry, the class is `internal`